src/lib/ui/Poper.module.css

```diff
@@ -0,0 +1,11 @@
+.scroll::-webkit-scrollbar {
+    display: none;
+}
+
+.scroll {
+    overflow-y: scroll;
+    height: 32rem;
+
+    -ms-overflow-style: none;
+    scrollbar-width: none;
+}
```

Comment:
this's not a shared behavior of the `Poper` component.  
Not all `Popers` with scroll will have this haight.  

Instead of making `Poper` scrollable make scrollable it's content